examples/cifar10/pt/learners/cifar10_learner.py

```diff
@@ -130,7 +133,7 @@ def initialize(self, parts: dict, fl_ctx: FLContext):
         # can be replaced by a config-style block
         self.device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
         self.model = ModerateCNN().to(self.device)
-        self.optimizer = optim.SGD(self.model.parameters(), lr=self.lr, momentum=0.9)
+        self.optimizer = optim.SGD(self.model.parameters(), lr=self.lr, momentum=0.9, weight_decay=0.0, nesterov=True)
```

Comment:
why do we change the nesterov from default False value to True?

Reply:
We followed the implementation in [Mostafa _et al._](https://arxiv.org/abs/1912.13075). Enabling the Nesterov momentum can stabilize the training.